tests/incr_test.go

```diff
@@ -0,0 +1,70 @@
+package tests
+
+import (
+	"fmt"
+	"sync"
+	"testing"
+)
+
+func TestINCRWhenKeyAlreadyExists(t *testing.T) {
+	var wg sync.WaitGroup
+	go runTestServer(&wg)
+
+	conn := getLocalConnection()
+
+	cmd := fmt.Sprintf("SET key %s", "1")
+	fireCommand(conn, cmd)
+
+	cmd = fmt.Sprintf("INCR key")
+
+	value := fireCommand(conn, cmd)
+
+	if value.(int64) != 2 {
+		t.Fail()
+	}
+
+	fireCommand(conn, "ABORT")
+	wg.Wait()
+}
+
+func TestINCRWhenKeyDoesNotExist(t *testing.T) {
+	var wg sync.WaitGroup
+	go runTestServer(&wg)
+
+	conn := getLocalConnection()
+
+	cmd := fmt.Sprintf("INCR key")
+
+	value := fireCommand(conn, cmd)
+
+	if value.(int64) != 1 {
+		t.Fail()
+	}
+
```

Comment:
1. You can merge all of these test cases in a single unit test.
2. Refer to https://github.com/DiceDB/dice/blob/master/core/bytelist_test.go. It will suggest how to write tests in Go.

Reply:
@arpitbbhayani I've addressed the review comments please have a look